charts/stable/synapse/questions.yaml

```diff
@@ -354,6 +345,16 @@ questions:
           schema:
             type: boolean
             default: false
+            show_subquestions_if: true
+            subquestions:
+              - variable: sharedSecret
+                label: Coturn Shared Secret
+                description: The shared secret to compute passwords for the TURN server
+                schema:
+                  type: string
+                  default: ""
+                  private: true
```

Comment:
Let's make it required, as it will otherwise will fail, as the _secret.tpl will need to have a value if that's coturn is enabled.

(if it's undersubquestions, and it's "required", only when the "parent" question is enabled this will be required